src/module_recipe.rs

```diff
@@ -106,33 +114,87 @@ impl Recipe {
             warn!("Running locally");
             tags.push(format!("{image_version}-local"));
         }
-        info!("Finished generating tags!");
+        debug!("Finished generating tags!");
         debug!("Tags: {tags:#?}");
         tags
     }
+
+    /// # Parse a recipe file
+    /// #
+    /// # Panics
+    /// #
+    /// # Errors
+    pub fn parse<P: AsRef<Path>>(path: &P) -> anyhow::Result<Self> {
+        let file_path = if Path::new(path.as_ref()).is_absolute() {
+            path.as_ref().to_path_buf()
+        } else {
+            std::env::current_dir().unwrap().join(path.as_ref())
+        };
+
+        let recipe_path = fs::canonicalize(file_path).unwrap();
```

Comment:
```suggestion
    /// # Errors
    pub fn parse<P: AsRef<Path>>(path: &P) -> anyhow::Result<Self> {
        let file_path = if Path::new(path.as_ref()).is_absolute() {
            path.as_ref().to_path_buf()
        } else {
            std::env::current_dir()?.join(path.as_ref())
        };

        let recipe_path = fs::canonicalize(file_path)?;
```
We really should try to avoid using unwrap wherever possible.